test/print_test.cpp

```diff
@@ -0,0 +1,35 @@
+#include <vector>
+
+#if COMMON_ROBOTICS_UTILITIES__SUPPORTED_ROS_VERSION == 2
+#include <geometry_msgs/msg/point.hpp>
+#elif COMMON_ROBOTICS_UTILITIES__SUPPORTED_ROS_VERSION == 1
+#include <geometry_msgs/Point.h>
+#else
+#error "Undefined or unknown COMMON_ROBOTICS_UTILITIES__SUPPORTED_ROS_VERSION"
+#endif
+
+#include <common_robotics_utilities/print.hpp>
+
+#include <gtest/gtest.h>
+
+namespace common_robotics_utilities
+{
+namespace print_test
+{
+GTEST_TEST(PrintTest, CanPrintROSMessages)
+{
+#if COMMON_ROBOTICS_UTILITIES__SUPPORTED_ROS_VERSION == 2
+  using Point = geometry_msgs::msg::Point;
+#elif COMMON_ROBOTICS_UTILITIES__SUPPORTED_ROS_VERSION == 1
+  using Point = geometry_msgs::Point;
+#endif
+  (void)print::Print(std::vector<Point>());
```

Comment:
Done in https://github.com/calderpg/common_robotics_utilities/pull/14/commits/3303e6861b7d7009132e66209d8eb0b6370204e1. Arguably, the output isn't great. `to_yaml` uses block (i.e. multiline) style instead of flow (i.e. inline) style and messes up with `Print()`.